db/migrate/20171001043440_create_games.rb

```diff
@@ -0,0 +1,12 @@
+class CreateGames < ActiveRecord::Migration[5.0]
+  def change
+    create_table :games do |t|
+      t.integer :id
+      t.integer :white_player_id
```

Comment:
`t.integer :white_player_id, index: true`

Add indexes for white_player_id and black_player_id so that look up will be fast